src/pages/components/Achievements.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react' ;
+import { VerticalTimeline, VerticalTimelineElement } from 'react-vertical-timeline-component';
+import 'react-vertical-timeline-component/style.min.css';
+import { Grid} from '@material-ui/core'
+import * as icons from '@material-ui/icons'
+
+
+export default function Achievements()
+{
+    
+    let  TuringHutAchievements  = [
+        {
+            title:"Title",
+            tagline:"tagline...",
+            description:"Description...",
+            date:"20 Jul 2020",  
+            imageAddress:"https://upload.wikimedia.org/wikipedia/en/d/d3/Hermione_Granger_poster.jpg"
+        },
+        {
+            title:"ACM ICPC 2019",
+            tagline:"tagline...",
+            description:"Description...", 
+            date:"20 Jul 2020", 
+            imageAddress:"https://upload.wikimedia.org/wikipedia/en/d/d3/Hermione_Granger_poster.jpg"
+        },
+        {
+            title:"ACM ICPC 2019",
+            tagline:"tagline...",
+            description:"Description...",
+            date:"20 Jul 2020",  
+            imageAddress:"https://upload.wikimedia.org/wikipedia/en/d/d3/Hermione_Granger_poster.jpg"
+        }
+    ]
```

Comment:
Isolate all of the Data and put them in a JSON file.

Reply:
Okay,